apps/sap-commerce-cloud/package.json

```diff
@@ -20,13 +20,12 @@
     "react-dom": "^17.0.1"
   },
   "scripts": {
-    "start": "cross-env BROWSER=none react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "test:ci": "CI=true react-scripts test",
+    "start": "vite",
+    "build": "vite build",
+    "test": "vitest --logHeapUsage --coverage",
```

Comment:
another thing I copied from MS teams that I'm not sure we need. Most of the other things I copied from online migration tutorials.